src/main/resources/webapp/js/components/ClustersComponent.js

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var ClustersComponent = React.createClass({
+  displayName: "ClustersComponent",
+
+  render: function () {
+    return(
+      <h1>Clusters</h1>
+   )}
+
+});
+
+module.exports = ClustersComponent;
```

Comment:
I don't really get the purpose of the clusters page.  Is this something you need for future work?